utils/default_config.go

```diff
@@ -40,6 +40,8 @@ func createConfigFromFlags(ctx *cli.Context) *Config {
 		CPUProfilePerInterval:  getFlagValue(ctx, CpuProfilePerIntervalFlag).(bool),
 		Cache:                  getFlagValue(ctx, CacheFlag).(int),
 		CarmenSchema:           getFlagValue(ctx, CarmenSchemaFlag).(int),
+		CarmenCheckpointInterval: getFlagValue(ctx, CarmenCheckpointInterval).(int),
+		CarmenCheckpointPeriod:   getFlagValue(ctx, CarmenCheckpointPeriod).(int),
```

Comment:
nit: keep alphabetical order 
```suggestion
		CarmenCheckpointInterval: getFlagValue(ctx, CarmenCheckpointInterval).(int),
		CarmenCheckpointPeriod:   getFlagValue(ctx, CarmenCheckpointPeriod).(int),
		CarmenSchema:                   getFlagValue(ctx, CarmenSchemaFlag).(int),
```